Docs/Book/RDKit_Book.rst

```diff
@@ -324,6 +324,65 @@ Here are some examples:
   ((0,), (2,), (3,), (4,), (5,))
 
 
+SMARTS Reference
+----------------
+
+*Note* that the text versions of the tables below include some backslash characters to
+escape special characters. This is a wart from the documentation system we are using.
+Please ignore those characters.
+
+**Atoms**
+
+=========  =========================================  ===============  ======  =========
+Primitive                  Property                   "Default value"  Range?    Notes
+=========  =========================================  ===============  ======  =========
+a          "aromatic atom"
+A          "aliphatic atom"
+D          "explicit degree"                          1                Y
+h          "number of implicit hs"                    >0               Y
+H          "total number of Hs"                       1
+r          "in SSSR ring of size"                     >0               Y
+R          "number of SSSR rings"                     >0               Y
+v          "total valence"                            1                Y
+x          "number of ring bonds"                     >0               Y
```

Comment:
should we be explicit that all "rings" are SSSR or just say it once at the top.

Reply:
It's probably worth clarifying that anyway, but it's not clear why it's relevant to this table.
The `number of ring bonds` is the same regardless of the ring definition and the others are already explicit that they look at SSSR